includes/class-wp-job-manager-rest-api.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * File containing the class WP_Job_Manager_REST_API.
+ *
+ * @package wp-job-manager
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Handles functionality related to the REST API.
+ *
+ * @since 1.33.0
+ */
+class WP_Job_Manager_REST_API {
+	/**
+	 * Sets up initial hooks.
+	 *
+	 * @static
+	 */
+	public static function init() {
+		add_action( 'rest_prepare_job_listing', array( __CLASS__, 'prepare_job_listing' ), 10, 2 );
```

Comment:
This should be a filter

Reply:
406ca3ed47dd98ba0283945acf6590cd2a389b9b